tests/php/Security/MemberTest.php

```diff
@@ -402,6 +404,160 @@ public function testIsPasswordExpired()
         $this->assertFalse($member->isPasswordExpired());
     }
 
+    public function testAdminCanRequirePasswordChangeOnNextLogIn()
+    {
+        /** @var Member&MemberExtension $targetMember */
+        $targetMember = $this->objFromFixture(Member::class, 'someone');
+        $this->logInWithPermission('ADMIN');
+        $field = $targetMember->getCMSFields()->dataFieldByName('RequiresPasswordChangeOnNextLogin');
+        $this->assertNotNull($field);
+    }
+
+    public function testUserCannotRequireTheirOwnPasswordChangeOnNextLogIn()
+    {
+        /** @var Member&MemberExtension $targetMember */
+        $targetMember = $this->objFromFixture(Member::class, 'someone');
+        $this->logInAs($targetMember);
+        $field = $targetMember->getCMSFields()->dataFieldByName('RequiresPasswordChangeOnNextLogin');
+        $this->assertNull($field);
```

Comment:
```suggestion
        $this->assertNull($field);
        $this->assertInstanceOf(CheckboxField::class, $field, 'The field should be an instance of CheckboxField::class');
```

Reply:
It can't be both null _and_ an instance of `CheckboxField`

Reply:
I assume you mean to add that in the test above, which checks if the admin can view the checkbox.

Reply:
Yeah, you are right :)))